panel/widgets/file_selector.py

```diff
@@ -3,23 +3,24 @@
 directories on the server.
 """
 import os
-
 from collections import OrderedDict
 from fnmatch import fnmatch
+from typing import AnyStr, List, Tuple, Union
 
 import param
+from _typeshed import StrPath
 
 from ..io import PeriodicCallback
 from ..layout import Column, Divider, Row
-from ..viewable import Layoutable
 from ..util import fullpath
+from ..viewable import Layoutable
 from .base import CompositeWidget
 from .button import Button
 from .input import TextInput
 from .select import CrossSelector
 
 
-def scan_path(path, file_pattern='*'):
+def _scan_path(path: StrPath, file_pattern='*') -> Tuple[List[str], List[str]]:
```

Comment:
I made this function private. Its only used in this module.

Reply:
Sounds good!